packages/ra-input-rich-text/src/index.tsx

```diff
@@ -125,14 +160,6 @@ RichTextInput.propTypes = {
     label: PropTypes.string,
     options: PropTypes.object,
     source: PropTypes.string,
-    toolbar: PropTypes.oneOfType([
```

Comment:
I removed that because `Quill` should validate it's options itself, IMO it's a good practice because leaving that breaks the composition.